fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersions.java

```diff
@@ -98,8 +98,11 @@ public FedoraVersions(final String path, final String label, final String pathIn
         this.pathIntoVersion = pathIntoVersion;
     }
 
+    /**
+     * Initialization method. Clients don't invoke this.
+     */
     @PostConstruct
-    private void postConstruct() {
+    public void postConstruct() {
```

Comment:
The @PostConstruct annotation works if the method is `private` or `public`. I would suggest leaving this as `private` and I can flag the Sonar issue as a `false positive`.